content/languages/scala/scalatest.md

```diff
@@ -7,6 +7,43 @@ tags:
 
 # ScalaTest
 
+## Basic Setup
+
+### Solution
+
+```scala
+object Adder {
+  def add(a: Int, b: Int) = a + b
```

Comment:
It can be a top level function in Scala 3, so it probably should be in the example?

Reply:
Fixed. Do you mind if I invite you for more reviews like this in the future?